nilearn/conftest.py

```diff
@@ -74,14 +75,17 @@ def close_all():
     yield
     if matplotlib is not None:
         import matplotlib.pyplot as plt
-        plt.close('all')  # takes < 1 us so just always do it
+
+        plt.close("all")  # takes < 1 us so just always do it
 
 
 def pytest_collection_modifyitems(items):
-    # numpy changed the str/repr formatting of numpy arrays in 1.14.
-    # We want to run doctests only for numpy >= 1.14.Adapted from scikit-learn
-    if _compare_version(np.__version__, '<', '1.14'):
-        reason = 'doctests are only run for numpy >= 1.14'
+    """Run doctests only for numpy >= 1.14.Adapted from scikit-learn.
+
+    numpy changed the str/repr formatting of numpy arrays in 1.14.
+    """
+    if _compare_version(np.__version__, "<", "1.14"):
```

Comment:
Do we really need this ?
I think we are only supporting 1.19 and above these days.

Reply:
I would say no given our setup.cfg says: numpy>=1.19.0

Want me to remove it as part of this PR?